From 34ff7ee2731cc5869caea6a28a89018d68afceed Mon Sep 17 00:00:00 2001
From: Rajesh Bhagat <rajesh.bhagat@nxp.com>
Date: Thu, 27 Apr 2017 12:11:48 +0530
Subject: [PATCH 17/26] arm64: fsl: la1575a: Add DTSI nodes for rtc in i2c node

Add DTSI nodes for rtc in i2c node for supporting date
command.

Signed-off-by: Rajesh Bhagat <rajesh.bhagat@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-la1575a-rdb.dts | 23 +++++++++++++++++++++++
 1 file changed, 23 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-la1575a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-la1575a-rdb.dts
index aaf1c30..82e6a44 100644
--- a/arch/arm64/boot/dts/freescale/fsl-la1575a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-la1575a-rdb.dts
@@ -94,3 +94,26 @@
 &rf5 {
 	status = "okay";
 };
+
+&i2c0 {
+	status = "okay";
+	pca9547@77 {
+		compatible = "philips,pca9547";
+		reg = <0x77>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		i2c@3 {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x3>;
+
+			rtc@51 {
+				compatible = "nxp,pcf2129";
+				reg = <0x51>;
+				/* IRQ10_B */
+				interrupts = <0 150 0x4>;
+			};
+		};
+	};
+};
-- 
2.7.4

